### PR TITLE
Setup steps for Cilium CNI

### PR DESCRIPTION
**Pull request by ackerman80**
Link to guide for setting up Cilium CNI with kubeadm.

### CODE REVIEW

**Comment by ackerman80** [comment]
Signed the CLA as a employee contributor part of the Covalent CNCF team

**Comment by neolit123** [comment]
@ackerman80 

thank you for the PR.
please squash the commits into a single one.

i'm getting the following message when i try to open the linked website:
error

i doubt that this will be accepted as is!

/assign @Bradamant3 @tengqm 
/cc @xiangpengzhao @dixudx 
/cc @kubernetes/sig-cluster-lifecycle-pr-reviews

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -277,8 +277,14 @@ For Canal to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
 ```
+{{% /tab %}}
+
+{{% tabe name="Cilium" %}}
+
+Cilium provides networking, L3/L4 network security and L7 (API-aware) security. Cilium is deployed as a CNI plugin and supports the Kubernetes Network Policy specifications. Please consult the [Cilium Kubernetes Install Guide](https://docs.cilium.io/en/stable/kubernetes/install/) for complete steps including RBAC settings, etcd access config and Kubernetes version specific daemonset configuration files.
```
Comment (neolit123) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: `daemonset` -> `DaemonSet`

**Comment by xiangpengzhao** [comment]
For the link issue, I guess "docs.cilium.io" is using "readthedocs.io" framework and its certificate then the browser can't verify the cert for "docs.cilium.io".
/cc @zacharysarah @steveperry-53 @Bradamant3

**Comment by ackerman80** [comment]
I've changed the link to https://cilium.readthedocs.io/en/stable/kubernetes/install/  as the certificate is for *.readthedocs.io should work. Also squashed the commits and made the other suggested edits. cc @xiangpengzhao @neolit123

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -277,6 +277,11 @@ For Canal to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
 ```
+{{% /tab %}}
+
+{{% tabe name="Cilium" %}}
+
+Cilium provides networking, L3/L4 network security and L7 (API-aware) security. Cilium is deployed as a CNI plugin and supports the Kubernetes Network Policy specifications. Please consult the [Cilium Kubernetes Install Guide](https://cilium.readthedocs.io/en/stable/kubernetes/install/) for complete steps including RBAC settings, etcd access configuration and Kubernetes version specific daemonset configuration files.
```
Comment (neolit123) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: `daemonset` -> `DaemonSet`

Comment (neolit123) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: other than that LGTM but would leave the approval to others.

**Comment by ackerman80** [comment]
Signed the CLA as a employee contributor part of the Covalent CNCF team

**Comment by neolit123** [comment]
@ackerman80 
if you have issues with the CLA contact the CNCF:
info@cncf.io
(or via a help desk if they have one.)

**Comment by ackerman80** [comment]
Yup sent a note to  helpdesk@rt.linuxfoundation.org

**Comment by neolit123** [comment]
thank for addressing the CLA issue.
this is LGTM, waiting on feedback from the SIG and we can possibly merge it.

**Comment by zacharysarah** [comment]
@cody-clark Looks like we need to fast-track your org membership!

**Comment by ackerman80** [comment]
@xiangpengzhao @neolit123 @zacharysarah is there something blocking to get this merged?

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 3]
```diff
@@ -277,6 +277,11 @@ For Canal to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
 ```
+{{% /tab %}}
+
+{{% tabe name="Cilium" %}}
```
Comment (xiangpengzhao) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: Netlify failed with this error:
```
11:28:05 PM: ERROR 2018/08/10 15:28:05 Unable to locate template for shortcode "tabe" in page "docs/setup/independent/create-cluster-kubeadm.md"
```

I think it's due to a typo here: `s/tabe/tab`

**Comment by ackerman80** [comment]
@xiangpengzhao fixed the typo. thanks for catching that

**Comment by xiangpengzhao** [comment]
/lgtm
Let other guys give a final approval if no more change needed :)

**Comment by neolit123** [comment]
a more detailed PR was submitted for Cilium here.
@ackerman80 please combine your efforts with that PR submitter and review his changes.
https://github.com/kubernetes/website/pull/9889
/close

**Pull request closed without merging**